oci-artificial-intelligence/ai-language/translation/translation.md

```diff
@@ -0,0 +1,122 @@
+    ```
+    ```
+
+    <copy>El Seattle Sounders Football Club anunció recientemente que estaba buscando un socio tecnológico para proporcionar una solución confiable, escalable y segura que pudiera ingerir, procesar y almacenar datos de juegos y jugadores. </copy>
+    ```
+    ```
+    <copy>En 2020, des personnes dans le monde entier ont commencé à travailler à distance en raison de la pandémie de COVID-19. Par conséquent, les outils collaboratifs tels que la vidéoconférence, les e-mails et les discussions sont devenus essentiels, car ils permettent aux employés d'effectuer leur travail à domicile. </copy>
+    ```
+
+3. **Click Translate:** 
+
+    You can translate text selecting the target language from the drop down menu and by clicking Translate button.
+    
+    ![OCI Language Text Analytics - Translate button](./images/translate-button.png " ")
+
+4. **Viewing Results:**
+
+    After you translate your text, the Language service displays the translated text.
+
+    ![Translation result](./images/translate-result.png " ")
+
+
+## **TASK 2:** Translate Text with Python SDK
+
+For setup, please refer Lab 1
+
+Follow below steps to run Python SDK:
+
+#### 1. Download Python Code.
+
+```Python
+<copy>
+import oci
+
+ai_client = oci.ai_language.AIServiceLanguageClient(oci.config.from_file(),
+                                                    service_endpoint="https://language.aiservice-preprod.us-ashburn-1.oci.oraclecloud.com")
+
+key1 = "doc1"
+key2 = "doc2"
+text1 = "The Indy Autonomous Challenge is the worlds first head-to-head, high speed autonomous race taking place at the Indianapolis Motor Speedway"
+text2 = "OCI will be the cloud engine for the artificial intelligence models that drive the MIT Driverless cars."
+
+doc1 = oci.ai_language.models.TextDocument(key=key1, text=text1, language_code="en")
+doc2 = oci.ai_language.models.TextDocument(key=key2, text=text2, language_code="en")
+documents = [doc1, doc2]
+
+compartment_id = "ocid1.tenancy.oc1..aaaaaaaaih4krf4od5g2ym7pffbp6feof3rx64522aoxxvv3iuw3tam6fvea" #TODO Provide your compartmentId here
+
+batch_language_translation_details = oci.ai_language.models.BatchLanguageTranslationDetails(documents=documents, compartment_id=compartment_id, target_language_code="de")
+output = ai_client.batch_language_translation(batch_language_translation_details)
+print(output.data)
+</copy>
+```
+
+Download [code](./files/translationPythonSDK.py) file and save it your directory.
+
+#### 2. Execute the Code.
+Navigate to the directory where you saved the above file (by default, it should be in the 'Downloads' folder) using your terminal and execute the file by running:
+
+```
+<copy>python translationPythonSDK.py</copy>
+```
+#### 3. Result
```

Comment:
remove ###2 and ###3 as they are trivial

Reply:
Removed